pom.xml

```diff
@@ -164,6 +174,9 @@
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-jar-plugin</artifactId>
           <version>2.4</version>
+          <configuration>
+            <outputDirectory>lib</outputDirectory>
```

Comment:
I'm not sure we want to replace the output location of the jar. Most Maven users expect the build jar to be in `/target`

I'd rather have an extra copy step.